pinot-segment-local/src/main/java/org/apache/pinot/segment/local/startree/v2/builder/MultipleTreesBuilder.java

```diff
@@ -115,13 +126,38 @@ public MultipleTreesBuilder(@Nullable List<StarTreeIndexConfig> indexConfigs, bo
     }
   }
 
+  private StarTreeIndexSeparator getSeparator()
```

Comment:
(minor) Annotate the return as `@Nullable`
```suggestion
  @Nullable
  private StarTreeIndexSeparator getSeparator()
```